src/GitHub.App/ViewModels/PullRequestDetailViewModel.cs

```diff
@@ -137,6 +137,12 @@ public class PullRequestDetailViewModel : BaseViewModel, IPullRequestDetailViewM
             DiffFile = ReactiveCommand.Create();
         }
 
+        void SubscribeOperationError(ReactiveCommand<Unit> command)
```

Comment:
Small nit: could you put this with the other private methods down below the public methods?